package.json

```diff
@@ -128,7 +128,7 @@
 	"private": true,
 	"repository": "git@github.com:3YOURMIND/kotti",
 	"scripts": {
-		"build:render.com": "yarn run watch:build-kotti-dependencies && yarn workspace @3yourmind/kotti-ui run build && yarn workspace @3yourmind/documentation run nuxt generate",
```

Comment:
@carsoli Can you explain why setting the webpack hash function wasn’t sufficient for this?

Reply:
couldn't